WORKSPACE

```diff
@@ -1,59 +1,22 @@
 workspace(name = "com_google_jsinterop_generator")
 
-maven_server(
-    name = "sonatype_snapshot",
-    url = "https://oss.sonatype.org/content/repositories/snapshots",
-)
-
-maven_jar(
-    name = "jsr305",
-    artifact = "com.google.code.findbugs:jsr305:3.0.1",
-)
-
-maven_jar(
-    name = "guava",
-    artifact = "com.google.guava:guava:21.0",
-)
-
-maven_jar(
-    name = "args4j",
-    artifact = "args4j:args4j:2.33",
-)
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-maven_jar(
-    name = "auto_value",
-    artifact = "com.google.auto.value:auto-value:1.4",
+http_archive(
+    name = "google_bazel_common",
```

Comment:
Note this is not an official project from the bazel team. There is no guaranty it will be maintained.

Reply:
It's a project from the Java Core Libraries Team (AFAICT), maintained for use by Google projects (and currently used by Flogger and Kythe)

@ronshapiro, could you weigh in as to whether it'd be a good idea to use it here (and in jsinterop-base) or not?

Reply:
@tbroyer is correct (and Dagger is on the way too).

@jDramaix for context, bazel-common is intended to make external bazel repos have a similar experience to using `//third_party` internally. No setup/workspace maintenance and consistent versioning. Additionally, we have some tools that build on top of it, such as the aforementioned `pom_file` and `javadoc_library`.

We maintain it and have interest in doing so actively.